main.go

```diff
@@ -164,10 +167,21 @@ func tryRebalance(ctx context.Context, r *regolancer, attempt *int) (err error,
 	routeCtxCancel()
 	for _, route := range routes {
 		log.Printf("Attempt %s, amount: %s (max fee: %s sat | %s ppm )",
-			hiWhiteColorF("#%d", *attempt), hiWhiteColor(amt), formatFee(fee), formatFeePPM(amt*1000, fee))
+			hiWhiteColorF("#%d", *attempt), hiWhiteColor(amt), formatFee(fee), formatFeePPM(amt*1000-fee, fee))
```

Comment:
This change is interesting. I'm personally not sure whether we should display the total amount with fees (that's being spent) or without (that we receive in target channel). Either way, if we change it here the success message should be changed too. I suggest we make it a separate PR, it doesn't affect the logic, only the presentation.

Reply:
ohh sry, was making some changes on dev branch, that got through, will revert.

Reply:
Yeah, I think we should unify that in another PR, bc I am calculating the success message taking the fees into account, so its even not correct what I did, I should have added the fees to be consistent with the success message